packages/app/src/sandbox/eval/transpilers/coffee/coffee-worker.js

```diff
@@ -0,0 +1,30 @@
+import { buildWorkerError } from '../utils/worker-error-handler';
+
+self.importScripts(
+  `${process.env.CODESANDBOX_HOST || ''}/static/js/coffeescript.2.3.2.js`
```

Comment:
Would it be better to pull this from a CDN instead of including this always?

Reply:
Hmm, I think it doesn't really matter where we get it from. CDN works too, might be a bit faster even, but not noticeably I think.